### DISCUSSION TITLE
Bunch of suggestions / campaign mission issues

### DISCUSSION THREAD

**Discussion by anarkopsykotik** [FakeFishGames/Barotrauma#14723] [Ideas and suggestions]
So in my latest playthrough with a friend I have encountered multiple issues / room for improvement, and I will post my feedback here.

ID cards
- there should be a way to authorize someone to access safes. On a 2 man multiplayer campaign, I had the captain duplicate his id and I had to use it the whole campaign.

The faction reputation system. Once you're hostile, the stations security will ask you to leave then shoot you
- They ask you to leave, but then actually follow you into your sub and then shootout ensue if you have bots attacking intruders, annoying because they are ridiculously tough (simply make them hold together rather than come suicide to you would be a good idea, and make aoe weapons more useful. And they shouldnt be so tough, or have it be an option like invincible outpost). 
- You can still take missions with the outpost leader, but he will still proceed to shoot you afterward (and only after you have taken all missions)
- NPC giving random mission will still start their dialogue, but then instantly close it and run away from you if you killed some security (even by drowning without actually shooting and making them hostile on sight, all their names were still blue)
- Security is "thankfully" easy to dispose of by drowning them in the airlock between the sub and the station by opening the doors. As they're dumb as bricks, they wont actually manage to either close the doors or leave before pressure blow them up.
- I tried disguising myself with a breathing mask or crawler mask and using various IDs. Sadly it does not work as they still recognize you instantly. Wreck IDs are not viable anyway because they dont allow you to open station doors. But even passengers IDs wont work to fool them (including the ID of a coalition agent obtained in a counter stakeout mission). Pretty disappointing.
- Hostile stations should never be invulnerable even with the option ticked.
- outpost leader safe seem always empty
- there should be a way to conquer hostiles outpost for your faction

The mission of obtaining proof the the coalition blew up a station 
- only the guy who did the dialogues can open the safe, even if you have the card

The mission of freeing prisoners in the outpost
- prisoners are absolute morons who will run away randomly into the station
- ordering them to go to the sub will not work, as they will go and then instantly leave
- they should simply wait there or follow you by default until ordered otherwise
- one time had the bad surprise of a security guard waiting for me inside a locked side alongside one prisoner
- I had a very fun time freeing one of them in an hostile station by going from outside, breaking a wall to break into the cell, quickly giving a pressure stabilizer to the prisoner and bringing him back by swimming.
- But when I had the sootman prisoner, I had to open him the door so he could get his revenge, and the security spent a long time simply aiming at him while he went for his rampage. Thankfully he managed to do it and go back to the sub fine, I guess/hope he is invulnerable in that part.
- The prisoners were apparently offended by me "stealing" from the outpost security corpse (and probably anything from the outpost for that matter)

Mission of transporting passenger
- Giving ABX antibiotics to a passenger to heal his infected wound cause you to loose reputation and he turns hostile

Assassinating a guy in a outpost for the husk church
- if the station is hostile, as you cant disguise, you again have to clear out the whole security first.

Abandoned outpost
- nobody access card allow to open safe (both armory and outpost leader ones). A safe cracker would be neat.

Pets
- they love to suicide by staying in flames
- Had the husk container commit suicide by jumping down a ladder
- they love getting on you while you destroy a ballast flora with flamer
- they are often bugged when entering a station : will keep going left or right for some reason, and refuse to eat because they're busy going into a direction even if they're against a wall. Going to lobby and reloading the station fixes it.
- they prevent watering plants when on top of them which is annoying
- orange boy could drop better stuff / chitin deconstruction is mediocre

Nav console
- would be cool to be able to rename a button you added
- 3 buttons is a bit limited
- some integrated light/text indicators would be cool
- directional sonar it would be neat to be able to choose the size of the angle

Reloading
- cant reload if you hotbar is full, your character will refuse to reload rather than throw the empty mag away, pretty annoying

Creatures
- had some husk manage to bite me through a wall in a wreck

Some suggestions
Depleted coilgun ammo should pierce like piercing coilgun ammo to be worth it
Would be cool to have depleted railgun shells, depleted chaingun ammo, depleted harpoon, depleted handcannon rounds, depleted rifle/assault rifle/HMG rounds.
Making choose between depleted ammo and tool is sad, cause tools arent worth it compared to depleted smg ammo.
Explosive are too clunky to use, detonator cant be crafted and dont have inbuilt radio detonation, I wanted to use them to blow open alien ruins but its easier to use the axe
Flares were rumored to attract creatures, but in my experience they'd much rather eat you, doesnt seem to give you even a few second. Using the gun flashlight make them straight useless (things are not very dark anyway).
Would be cool to be able to remove vegetation (the annoying one slowing you down in caves) with knife (and gather plants more quickly)
Reward for killing treshers is pretty small. Tresher used to give adrenaline iirc but no more, now its just blood and divers remains.
Some buildup of molluscs on the hull slowing you/slowing turrets rotations/slowing the engine would be a neat way to force you to go out from time to time.
More passive creatures / only aggressive when you attack them or go close would be cool. Also more creatures giving various poisons status.
A creature "kidnapping" a player without outright killing him could be fun too.
Gun loaders should deteriorate a bit more and jam sometimes.
Welding fuel consumption have become minuscule it seems, i would increase it significantly. Oxygen have a nice balance now though.
Still want a "brace for impact" mechanic, holding to something to prevent big shock from hurting you (and big shock hurting you more/making boxes flying around dangerous)
The PUCS is just shit now. The fissile accelerator is kinda unusable against anything trying to go melee.
Radiation is a non issue unless you make it an issue yourself.
Makeshift shelve would be more useful if you could put ammo boxes in them.
Let us rewire components in the wall without needing to destroy the walls.
A mission where you need to take some people alive could be cool. The perk giving additional reward for it is not worth the hassle.
Could be cool to be able to see a bit further away. In caves the creatures always see you first and attack unless they're stuck.
A lifeline attaching you to the sub could be a cool way to retrieve you in case of problem in a cave
You should auto drop the cargo scooter when trying to equip a weapon
Being able to take control of bots in multi would be cool to be able to use some of their talents.
Having a railgun shell that can penetrate multiple target would be cool.
Passive sonar could be buffed a bit so you can see cave walls from further. But Im not sure staying passive really work anyway, you usually still get some aggro.
It would be cool if line of sight was actually directional and you couldnt see behind you.

**Comment by Ivanzyfer** [comment]
Hostile stations should not be allowed to dock to themselves at all, but should open fire to kill (but now they do not have guns to shoot), instead it should be possible to skip stations and sail on without docking. When approaching a station, a button should appear at the top so that you can move to the next location without connecting to the station.
Or should there be an option to storm the station and then capture it for your faction.

**Comment by soltannis** [comment]
Some really great stuff in here and hard to find any to disagree with. 
Totally right about radiation and missed potential with so much nuclear in the game. 
picturing spills from reactor, wrecks leaking, stations giving cleanup missions, transporting waste and so on.
